packages/fiori/src/SideNavigationGroup.ts

```diff
@@ -123,11 +123,7 @@ class SideNavigationGroup extends SideNavigationItemBase {
 	}
 
 	get belowGroupClassName() {
-		if (isInstanceOfSideNavigationGroup(this.previousElementSibling)) {
-			return "ui5-sn-item-group-below-group";
-		}
-
-		return "";
+		return this.previousElementSibling && isInstanceOfSideNavigationGroup(this.previousElementSibling) ? "ui5-sn-item-group-below-group" : "";
```

Comment:
That should be reworked in the future. Component shouldn't know about the surrounding context if it's not explicitly set by parent element or the developers.